plugins/modules/keycloak_group.py

```diff
@@ -69,6 +71,33 @@
             - A dict of key/value pairs to set as custom attributes for the group.
             - Values may be single values (e.g. a string) or a list of strings.
 
+    parents:
+        version_added: "6.3.0"
+        type: list
+        description:
+            - List of parent groups for the group to handle sorted top to bottom.
+            - Set this to create a group as a subgroup of another group or groups (parents).
+        elements: dict
+        suboptions:
+          id:
+            type: str
+            description:
+              - Identify parent by ID
+              - Needs less API calls than using I(name)
+              - A deep parent chain can be started at any point when first given parent is given as ID
+              - Note that in principle both ID and name can be specified at the same time
+                but current implementation only always use just one of them, with ID
+                being prefered
+          name:
+            type: str
+            description:
+              - Identify parent by name
+              - Needs more internal API calls than using I(id) to map names to ID's under the hood
+              - When giving a parent chain with only names it must be complete up to the top
+              - Note that in principle both ID and name can be specified at the same time
+                but current implementation only always use just one of them, with ID
+                being prefered
```

Comment:
```suggestion
              - Identify parent by ID.
              - Needs less API calls than using I(name).
              - A deep parent chain can be started at any point when first given parent is given as ID.
              - Note that in principle both ID and name can be specified at the same time
                but current implementation only always use just one of them, with ID
                being preferred.
          name:
            type: str
            description:
              - Identify parent by name.
              - Needs more internal API calls than using I(id) to map names to ID's under the hood.
              - When giving a parent chain with only names it must be complete up to the top.
              - Note that in principle both ID and name can be specified at the same time
                but current implementation only always use just one of them, with ID
                being preferred.
```

Reply:
Oh my, collected a lot of typos here :), should be fixed now.